### ISSUE TITLE
Custom tooltip

### ISSUE THREAD

**Issue by AfsalXPO** [ag-grid/ag-grid#2178]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ x ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Cell tool tip showing default 'title' attribute style. (black text in white overlay).

**Expected behavior**
 Custom tool tip in black overlay with pointer to cell.
We tried to integrate this like 
https://jsfiddle.net/AfsalOvingal/xcwj2pjL/1/ but when we scroll tool tip position is changing independent of cell position. 
Additional to this we need one pointer to point to the cell.
Expected result image :-
![tooltip](https://user-images.githubusercontent.com/32155733/35672373-58906520-0764-11e8-8613-37c46aa4869e.PNG)

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**
Tool tip with custom style

**Please tell us about your environment:**

* **ag-Grid version:** 13.1.1

* **Browser:**
All

* **Language:** all

**Comment by catahoc** [comment]
Hi! This is reproduced in latest ag-grid, so +1. We have the same issue: complex formatted tooltip. Cells have "overflow: hidden:", so tooltip only works with "position: fixed", but in that case it works badly with scrolling. We need to have either some css workaround to support formatted tooltips along with scrolling, or fine-grained solution aka "tooltipRendererFramework"
![image](https://user-images.githubusercontent.com/2704650/37021108-74159f06-212f-11e8-9d88-6a0206d8af26.png)

**Comment by mwhitworth-auro** [comment]
Agree, we're seeing this issue too.

**Comment by kinamartinez** [comment]
Hello, I would like to know how do you solve this problem? I have the same problem.. Thank you in advance for your answer

**Comment by makinggoodsoftware** [comment]
Hi,

Thank you for your email. We are aware of this issue and have a feature request in our backlog:

You can follow the status of feature requests, bugs and releases using our pipeline: 
https://www.ag-grid.com/ag-grid-pipeline/

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.



This is the reference and summary of the relevant issue:

AG-1617 | Allow complex tooltip/renderer components in ag-grid to stick to the row as the user scrolls
-- | --

**Comment by makinggoodsoftware** [comment]
Note that this is high in our radar, we don't think this to be on the next release, but we are hoping to allow custom tooltips that stick to rows to be available in the next 2/3 releases

Thanks for all the feedback

**Comment by nechuminwebit** [comment]
fgg

**Comment by havietduc91** [comment]
Hi @makinggoodsoftware , firstly thank you for awesome library and what's status of issue  `AG-1617`? I don't see it when I filter `1617` on pipeline page, https://www.ag-grid.com/ag-grid-pipeline/

Thank you first!

**Comment by JabbyPanda** [comment]
Custom tooltips are available since ag-grid 20.1 

https://medium.com/ag-grid/introducing-version-20-1-of-ag-grid-d19753b99d5f

**Comment by shijinmk2006** [comment]
Am using Ag grid with vue+typescript with custom tool-tip component 

am getting this error when i use tool-tip feature. any [help]
![image](https://user-images.githubusercontent.com/4848814/91008458-11b7d100-e5fc-11ea-9d27-1629c18e57f3.png)
